frontend/src/utils/configs.js

```diff
@@ -1,5 +1,6 @@
 //------- CONNECTION OR SETUP CONFIGS --------//
-const URI_USER_SVC = process.env.URI_USER_SVC || 'http://localhost:8000'
+const URI_USER_SVC =
```

Comment:
Environment variables in a react app must start with `REACT_APP` in order for `create-react-app` to read them at build time